app/views/govuk_component/docs/alpha_label.yml

```diff
@@ -0,0 +1,9 @@
+---
```

Comment:
Discussed with @dsingleton via other channels but summarising here. `---` is being added by the generator and seems to be a YAML thing called a [structure](http://www.yaml.org/spec/1.2/spec.html). 

> YAML uses three dashes (“---”) to separate directives from document content. This also serves to signal the start of a document if no directives are present.

Probably better leaving them in each file if it's output from the generator so people later on won't think the generator is being weird.